examples/observing_notebooks/CWFSAlignFocusScript.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<p>Maybe add a comment about how we should always use exposure times that are ~20+ seconds to ensure the doughnuts shape isn't dominated by atmospheric (dome) turbulence.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>